### PR TITLE
feat: :art: stop   repeat   request

### PR DESCRIPTION
**Pull request by 1linan**
## Description
阻止重复请求

## Checklist

- [ ] the pull request title describes what this PR does (not a vague title like `Update index.md`)
- [ ] the pull request targets the *default* branch of the repository (`develop`)
- [ ] the code follows the established code style of the repository
  - `npm run prettier:check` passes
  - `npm run eslint:check` passes
- [ ] tests are added for the changes I made (if any source code was modified)
- [ ] documentation added or updated
- [ ] I have run the project locally and verified that there are no errors

### Fixes

fixes #[issue number], fixes #[issue number]

### CODE REVIEW

File: src/pages/about/contract/index.tsx [hunk 1]
```diff
@@ -357,5 +357,8 @@ const MyContract: React.FC<MyContractProps> = (props) => {
 export default connect(({ User, Contract, loading }: ConnectState) => ({
   User,
   Contract,
-  qryLoading: loading.effects['Contract/getPageListOfChainCode']
+  qryLoading:
+    loading.effects['Contract/getPageListOfChainCode'] ||
+    loading.effects['Contract/installContract'] ||
```
Comment (Liyw-liyw) [inline src/pages/about/contract/index.tsx]: 安装合约和发布合约的effect赋给query loading干嘛

File: src/pages/about/enterprise-member/index.tsx [hunk 2]
```diff
@@ -414,5 +417,9 @@ export default connect(({ User, Layout, Member, loading }: ConnectState) => ({
   User,
   Layout,
   Member,
-  qryLoading: loading.effects['Member/getPageListOfCompanyMember']
+  qryLoading:
+    loading.effects['Member/getPageListOfCompanyMember'] ||
+    loading.effects['Member/setStatusOfLeagueCompany'] ||
```
Comment (Liyw-liyw) [inline src/pages/about/enterprise-member/index.tsx]: 同上，这些操作赋给query loading是做什么

File: src/pages/user/set-password/index.tsx [hunk 3]
```diff
@@ -224,9 +224,9 @@ const SetPassword: React.FC<SetPasswordProps> = (props) => {
       </Link>
     </div>
   );
-}
+};
 
 export default connect(({ User, loading }: ConnectState) => ({
   User,
-  loginLoading: loading.effects['User/login'],
+  loginLoading: loading.effects['User/changePassword']
```
Comment (Liyw-liyw) [inline src/pages/user/set-password/index.tsx]: 登录按钮的loading为何赋值为 修改密码的？

File: src/pages/about/contract/index.tsx (hunk 1 above)
Comment (1linan) [reply src/pages/about/contract/index.tsx]: qryLoading的true或false不是要根据调用的函数变吗

File: src/pages/user/set-password/index.tsx (hunk 3 above)
Comment (1linan) [reply src/pages/user/set-password/index.tsx]: 这个页，不是修改密码那页码？点击修改密码调用的是UserModule中changePassword函数，我就改成这个函数了

**Pull request merged**